src/yang/ietf-bgp-rib.yang

```diff
@@ -50,7 +46,9 @@ submodule ietf-bgp-rib {
 
      Authors: Mahesh Jethanandani (mjethanandani at gmail.com),
               Keyur Patel (keyur at arrcus.com),
-              Susan Hares (shares at ndzh.com)";
+              Susan Hares (shares at ndzh.com),
+              Jeffrey Haas (jhaas at pfrc.org).";
```

Comment:
Could you update everything to be jhaas@juniper.net?  pfrc.org is just my workaround for dealing with ietf mailing lists when work's email is... not optimal.

Reply:
Done